Dockerfile

```diff
@@ -1,9 +1,12 @@
 # This Dockerfile contains multiple targets.
 # Use 'docker build --target=<name> .' to build one.
 
+ARG ALPINE_VERSION=3.16.2
+ARG GO_VERSION
+
 # devbuild compiles the binary
 # -----------------------------------
-FROM docker.mirror.hashicorp.services/golang:1.17.6 AS devbuild
+FROM docker.mirror.hashicorp.services/golang:${GO_VERSION} AS devbuild
```

Comment:
Will this work with the CRT docker build? I think docker build may complain if the image tag isn't set.

Reply:
Good point, I thought I had successfully run `docker build --target=default .` to test it before pushing, but it did break CRT builds. I've defaulted it to `latest` now, just so that the Dockerfile is valid and can be parsed, but that latest tag isn't actually used in any Makefile targets or CRT builds.